### PR TITLE
fix indirect link (core libraries committee)

### PR DESCRIPTION
**Pull request by dschrempf**
Tiny PR. Fixes an indirect link to the Wiki and then to GitHub.

### CODE REVIEW

**Comment by tomjaguarpaw** [comment]
Thanks! We'll need to wait for one more approval.

**Review by TikhonJelvis — APPROVED** [review]

**Pull request merged**
